docs/GDI-SOP_charter.md

```diff
@@ -0,0 +1,126 @@
+
+## 6. Scope of GDI SOPs
+The GDI is not a monolithic entity but a network of existing and emerging infrastructures. While SOPs can encompass various aspects of a team, node, or even project, this repository is not intended to cover every process within all institutions under the GDI umbrella. Instead, it contains SOPs that are relevant to GDI-specific use cases and are implemented by the nodes forming the GDI network.
+
+For example, if a GDI node already functions as an archive with submitted datasets, the curation of these datasets, when outside the GDI scope, is not expected to follow a hypothetical "Dataset Curation SOP" from GDI.
+
+Therefore, it is the **responsibility of the GDI nodes to implement these SOPs when the use case pertains to GDI infrastructure**, governance, or data.
+
+## 7. GDI SOP Management
+Given the diverse fields that GDI SOPs may cover, a well-defined set of roles and responsibilities is crucial for ensuring seamless operations across the network. These roles are involved at every step of the SOP lifecycle, from development to node implementation. The most up-to-date definitions of roles and responsibilities can be found [here](./GDI-SOP_organisational-roles-and-responsibilities.md#6-roles-and-responsibilities-definitions).
+
+The process of adding a new SOP or editing an existing one is itself governed by established SOPs, such as the [GDI-SOP0007_SOP-template-creation.md](../sops/european-level/GDI-SOP0007_SOP-template-creation.md). **Repository maintainers must ensure that any changes are justified and consider their impact on both planned and existing GDI node features**. To facilitate this, the OC and SDPC include representatives from all GDI nodes across the three GDI pillars, ensuring that all are informed through mailing lists (see [GDI SOP Communication](#8-gdi-sop-communication)) about any SOP developments or changes. SOPs undergo rigorous review, approval, and authorization processes involving diverse GDI members to ensure quality and relevance.
+
+Flexibility is also essential, ensuring that **GDI SOPs are strict enough to be followed and maintained while allowing GDI nodes the flexibility to implement them as needed**. The responsibility for maintaining the integrity of these implementations (i.e. being _similar enough_ to the templates) lies with the members of each node who participate in the OC/SDPC. This accountability framework allows for quick and flexible node implementation without imposing excessive constraints from the central GDI governance. For more information, refer to the [Roles and Responsibilities](./GDI-SOP_organisational-roles-and-responsibilities.md#6-roles-and-responsibilities-definitions), [GH management](./GDI-SOP_github-management.md) documentation, and the [summary diagram](../README.md#summary-diagram).
```

Comment:
You can consider if you want to change the following sentence: "each node who participates" to "each node participating"... if you keep it as it is now, then you might change participate to participates. :-)

Reply:
I think in the phrase I'm referring to the members _who participate_ in the OC/SDPC. Since it's expected that OC/SDPC has participation from all nodes, I was trying to pinpoint the exact members in these committees, rather than the exact nodes in them (all, _a priori_)